modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -2707,20 +2707,19 @@
 
     <xsd:element name="fixed-frequency-scheduler" type="fixedSchedulerType" substitutionGroup="abstract-scheduler"/>
 
-    <xsd:element name="poll" type="pollMessageSourceType" substitutionGroup="abstract-message-source">
+    <xsd:element name="scheduler" type="schedulerMessageSourceType" substitutionGroup="abstract-message-source">
         <xsd:annotation>
             <xsd:documentation>
                 Polls the nested message processor for new messages
             </xsd:documentation>
         </xsd:annotation>
     </xsd:element>
-    <xsd:complexType name="pollMessageSourceType">
+    <xsd:complexType name="schedulerMessageSourceType">
         <xsd:complexContent>
             <xsd:extension base="abstractMessageSourceType">
                 <xsd:sequence>
                     <xsd:element ref="abstract-scheduler" minOccurs="0" maxOccurs="1"/>
-                    <xsd:element ref="abstract-poll-override" minOccurs="0" maxOccurs="1"/>
-                    <xsd:group ref="messageProcessorOrMixedContentMessageProcessor"/>
+                    <xsd:element ref="abstract-scheduler-override" minOccurs="0" maxOccurs="1"/>
                 </xsd:sequence>
                 <xsd:attribute name="frequency"  type="substitutableLong" default="1000">
```

Comment:
Make sense to clean things up and introduce a simple-scheduler child element with frequency, to avoid needing the exclusiveness checks to ensure `frequency` fails if used at same time as a scheduler